Gruntfile.js

```diff
@@ -59,6 +59,38 @@ module.exports = function(grunt) {
           {expand: true, cwd: 'test/fixtures/', src: ['**'], dest: './'}
         ]
       },
+      zipDoNotCreateEmptyArchiveTrue: {
+        options: {
+          doNotCreateEmptyArchive: true,
+          archive: function () {
+            return 'tmp/compress_test_files_empty_must_not_be_created_because_option_set_to_true.zip';
+          }
+        },
+        files: [
+          {expand: true, cwd: 'test/fixtures/', src: ['NotExistingFilePath.js']}
```

Comment:
Same code on other lines in this file. Should I refactor coding style in other lines?
Lines 51, 59, 105, 113, 121, 129 in 7feb42e rev.